actions/benchmarkActions.js

```diff
@@ -0,0 +1,40 @@
+import fetch from 'isomorphic-fetch';
+import { API_HOST } from './../actions';
+
+const url = '/api/benchmarks';
+
+const START = 'yasp/benchmark/START';
+const DONE = 'yasp/benchmark/DONE';
+const ERROR = 'yasp/benchmark/ERROR';
+
```

Comment:
ok, I used to use START, DONE, ERROR as action constant on my own react project.